mleap-spark/src/main/scala/org/apache/spark/ml/bundle/ops/classification/LogisticRegressionOpV21.scala

```diff
@@ -43,7 +43,7 @@ class LogisticRegressionOpV21 extends SimpleSparkOp[LogisticRegressionModel] {
         val coefficientMatrix = Matrices.dense(cmTensor.dimensions.head, cmTensor.dimensions(1), cmTensor.toArray)
         val lr = new LogisticRegressionModel(uid = "",
           coefficientMatrix = coefficientMatrix,
-          interceptVector = Vectors.dense(model.value("intercept").getTensor[Double].toArray),
+          interceptVector = Vectors.dense(model.value("intercept_vector").getTensor[Double].toArray),
```

Comment:
Can we leave this as "intercept" so we don't break our serialization format?

Reply:
@hollinwilkins in case of multinomial log regr, it is being stored as "intercept_vector", here is just the fix to load it back correctly.